src/features/products/screen/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FlatList, ScrollView, View } from 'react-native';
+
+import { useGetProducts } from 'network/queries/product-queries';
+
+import ProductItem from './productItem';
+import styles from './styles';
+
+const PurchasesScreen = () => {
+  const { data } = useGetProducts();
```

Comment:
You can destructure data to have the final object available directly from the query.

```suggestion
  const { data: { data } } = useGetProducts();
```

Reply:
this wont work as the useGetProducts could return Undefined